pysrc/juliacall/__init__.py

```diff
@@ -248,8 +248,34 @@ def jlstr(x):
                 "PYTHON_JULIACALL_HANDLE_SIGNALS=no."
             )
 
-init()
+    # Next, automatically load the juliacall extension if we are in IPython or Jupyter
+    CONFIG['autoload_ipython_extension'] = choice('autoload_ipython_extension', ['yes', 'no'])[0]
+    if CONFIG['autoload_ipython_extension'] in {'yes', None}:
+        try:
+            get_ipython = sys.modules['IPython'].get_ipython
+
+            if CONFIG['autoload_ipython_extension'] is None:
+                # Only let the user know if it was not explicitly set
+                print(
```

Comment:
Should maybe be a warning, so can be disabled using Python's warnings mechanisms?

Reply:
I'm ambivalent about this.

Reply:
If it is a warning, it feels like you *need* to set the environment variable (one way or the other), and ends up being a pain. When I have this set up I just want it to run automatically and not worry about it. A single-line print seems nice though.

I also tried `logger.info` but it's also a pain because you need to reconfigure logging defaults... I guess I think this is really what `print` is made for. The user can always set that env variable to turn off printing.